components/vf-chatbot-dialog/vf-chatbot-dialog.scss

```diff
@@ -0,0 +1,102 @@
+.vf-chatbot-dialog {
+  position: absolute; // Change from fixed to absolute
+  top: 0;
+  left: 0;
+  padding: 5%;
+  width: 90%;
+  height: 100%;
+  background: rgba(0, 0, 0, 0.5);
+  display: none;
+  justify-content: center;
+  align-items: center;
+  z-index: 9999; // Increase z-index to be above other modal elements
+
+  &__content {
+    background: white;
+    padding: 24px;
+    width: 100%;
+    max-width: 480px;
+    box-shadow: 0 4px 6px rgba(0, 0, 0, 0.1);
+    position: relative;
+    margin: auto;
+    animation: dialogFadeIn 0.2s ease-out;
+  }
```

Comment:
Body text could be easily used as existing vf-content / vf-text class, no need to duplicate it